docs/How-We-Work/Secrey-Key-Mgmt.md

```diff
@@ -0,0 +1,43 @@
+- a security breach in one or more of the project [tools](https://about.codecov.io/security-update/) that use the secret keys(*relatively speaking, this is more challenging to prevent*), or 
+- a team member with access to the secret keys leaks them (*this is usually unintentional*)
+
+## Mitigation steps
+Below are a list of steps the team is taking to help mitigate the incidence of secret key leakage and stay up-to-date on news that could potentially compromise the security state of our TDP system. This may be updated over time, as additional or alternative solutions are adopted. 
+
+### Current practices
+- Use `git status` terminal command before any commits and pushes to github repo. This should help detect any secret key files that have been modified prior to `git commit`. 
+- Prioritize updating [gitignore](https://git-scm.com/docs/gitignore) before submitting any follow-on PRs that involve use of files that include secret keys. This will prevent any files that contain secret keys from being committed to the repo. 
+- Secret keys are to be retrieved from cloud.gov for local development purposes. Cloud.gov is a platform that requires these keys, and the dev team has access to the keys stored in the dev environment space. Therefore, this is a more secure approach for retrieving keys than relying on team members to share keys across other platforms/tools.  
+- No production keys will be stored on local machines, since this is unnecessary for development work.  In the event of unintended secret key leakage, this would have no impact on the production environment. Only people who have access to the production space in Cloud.gov will have access to prod keys.
+- At a minimum, ACF and vendor Tech Leads should be subscribed to tech tools we use to stay up-to-date on news that could impact project security. This includes: CircleCi, CodeCov, GitHub, Cloud.gov, Docker
+- As part of ACF Tech Lead's periodic review of environment variables, secret key rotation will be coordinated.
+
+### Longer-term solutions to be implemented (these are housed under epic [#972](https://github.com/raft-tech/TANF-app/issues/972):
+- Automated tool will be added to detect secret keys before local developers can commit changes to project repo (#965) 
+- Add .dockerconfig file as an extra layer of security against accidentally leaking secrets as part of local development (#544)
+- Automated testing step will be added in CircleCI to check for secret keys and stop the CI process if keys are detected (#966)
+- `DJANGO_SECRET_KEY` is automatically generated for initial deployments to Cloud.gov. This will ensure that the key is not shared across any environments and never needs to be exposed to developers or stored outside of Cloud.gov (#967)
```

Comment:
```suggestion
- `DJANGO_SECRET_KEY` is automatically generated for initial deployments to Cloud.gov. This will ensure that the key is not shared across any environments and never needs to be exposed to developers or stored outside of Cloud.gov ([#967](https://github.com/raft-tech/TANF-app/issues/967))
```